app/models/assignment.rb

```diff
@@ -2,6 +2,7 @@
 
 class Assignment < ApplicationRecord
   validates :name, length: { minimum: 1 }
+  validates :grading_scale, inclusion: { in: %w(percent), message: "needs to be fixed at 1-100 for passing the grade back to LMS" }, if: :lti_integrated?
```

Comment:
Make the error message translatable

Reply:
I have kept them intentionally hardcoded because our branch will surely get a conflict with `master` for the i18n so finally before merging to master we can resolve all of them at a time